api/app/signals/apps/signals/models/routing_expression.py

```diff
@@ -8,7 +8,7 @@ def update_routing(self, instance, data):
         department = data['_department']['id']
         order = data.get('order', None)
         if not hasattr(instance, 'routing_department'):
-            instance.routing_department = RoutingExpression.objects.create(
+            instance.routing_department = RoutingExpression.actions.create(
```

Comment:
See above comment on `actions` and `objects` managers.

Reply:
check